### PR TITLE
[FLINK-5474] [docs] Extend DC/OS documentation

### PR DESCRIPTION
**Pull request by tillrohrmann**
State where you can find out the job manager's address and how to install the DC/OS CLI.

### CODE REVIEW

File: docs/setup/mesos.md [hunk 1]
```diff
@@ -92,18 +92,25 @@ Universe. In the search prompt, just search for Flink.
 **Note**: At the time of this writing, Flink was not yet available in the
 Unvierse. Please use the following workaround in the meantime:
```
Comment (tzulitai) [inline docs/setup/mesos.md]: This isn't part of your change, but just spotted this typo :-) : "Universe"

File: docs/setup/mesos.md [hunk 2]
```diff
@@ -92,18 +92,25 @@ Universe. In the search prompt, just search for Flink.
...
-   `dcos package repo add --index=0 dev-universe http://universe.marathon.mesos:8085/repo`
+   `./dcos package repo add --index=0 dev-universe http://universe.marathon.mesos:8085/repo`
 
-3. Install Flink through the Universe page or using the `dcos` command:
+4. Install Flink through the Universe page or using the `dcos` command:
    
-   `dcos package install flink`
+   `./dcos package install flink`
+
+In order to execute a Flink job on a DC/OS hosted Flink cluster, you first have to find out the address of the launched job manager.
```
Comment (tzulitai) [inline docs/setup/mesos.md]: job manager --> I think we mostly use "JobManager" in docs

File: docs/setup/mesos.md [hunk 3]
```diff
@@ -92,18 +92,25 @@ Universe. In the search prompt, just search for Flink.
...
+The job manager address can be found out by opening the Flink service, going to *Job Manager* and then using the address specified under `jobmanager.rpc.address` and `jobmanager.rpc.port`.
```
Comment (tzulitai) [inline docs/setup/mesos.md]: "JobManager"

File: docs/setup/mesos.md (hunk 1 above)
Comment (tillrohrmann) [inline docs/setup/mesos.md]: Will correct it.

File: docs/setup/mesos.md (hunk 2 above)
Comment (tillrohrmann) [inline docs/setup/mesos.md]: I'll change it.

File: docs/setup/mesos.md (hunk 3 above)
Comment (tillrohrmann) [inline docs/setup/mesos.md]: Same here.

**Comment by tillrohrmann** [comment]
Thanks for the review @tzulitai. I'll address your comments and will then merge this PR.

**Pull request closed without merging**